SendWithUs.Client/Requests/SendRequest.cs

```diff
@@ -22,9 +22,7 @@ namespace SendWithUs.Client
 {
     using System;
     using System.Collections.Generic;
-    using System.Linq;
     using Newtonsoft.Json;
-    using Newtonsoft.Json.Linq;
 
     /// <summary>
     /// Represents the data necessary to make API requests to send email messages.

```

Comment:
`SendRequest<TData>` is purely a convenience for developers that want a strongly-typed `Data` property. It really adds no value, since the code that created the `SendRequest` already knows the type of the value set on `Data`, and that code has little reason to invoke the getter. I'm more inclined to drop `SendRequest<TData>` altogether than to refactor it.